src/main/java/uk/gov/justice/digital/job/DataHubJob.java

```diff
@@ -94,8 +94,8 @@ private void batchProcessor(JavaRDD<byte[]> batch) {
                     val structuredLoadDataFrame = structuredZone.processLoad(spark, dataFrameForTable, table);
                     curatedZone.processLoad(spark, structuredLoadDataFrame, table);
 
-                    val structuredCDCDataFrame = structuredZone.processCDC(spark, dataFrameForTable, table);
-                    curatedZone.processCDC(spark, structuredCDCDataFrame, table);
+                    val structuredIncrementalDataFrame = structuredZone.processCDC(spark, dataFrameForTable, table);
```

Comment:
there is not need for procesLoad and processCDC methods. we can call process() method directly passing the boolean flag to identify isCDC

Reply:
This is now updated. Thanks